services/src/test/java/org/exoplatform/news/NewsTargetingImplTest.java

```diff
@@ -35,17 +36,19 @@ public void shouldReturnTargets() {
     Metadata sliderNews = new Metadata();
     sliderNews.setName("sliderNews");
     sliderNews.setCreatedDate(100);
-    HashMap<String, String> sliderNewsProperties = new HashMap<>();
-    sliderNewsProperties.put("label", "slider news");
+    List<MetadataProperties> sliderNewsProperties = new LinkedList<>();
```

Comment:
to be updated according to social [Social PR](https://github.com/Meeds-io/social/pull/1103)